src/AltinnCore/UnitTest/AltinnCore.UnitTest.csproj

```diff
@@ -32,18 +35,21 @@
   </PropertyGroup>
   
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DocumentationFile>bin\Debug\netcoreapp2.1\AltinnCore.Common.xml</DocumentationFile>
+    <DocumentationFile>bin\Debug\netcoreapp2.2\AltinnCore.Common.xml</DocumentationFile>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Manatee.Json" Version="10.1.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
-    <PackageReference Include="Moq" Version="4.10.0" />
-    <PackageReference Include="NJsonSchema" Version="9.13.3" />
-    <PackageReference Include="NJsonSchema.CodeGeneration.CSharp" Version="9.13.3" />
+    <PackageReference Include="Manatee.Json" Version="10.1.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
+    <PackageReference Include="Moq" Version="4.10.1" />
+    <PackageReference Include="NJsonSchema" Version="9.13.37" />
+    <PackageReference Include="NJsonSchema.CodeGeneration.CSharp" Version="9.13.37" />
     <PackageReference Include="XmlDiffPatch.Core" Version="1.0.1" />
-    <PackageReference Include="xunit" Version="2.3.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
     <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
```

Comment:
just wondered if this package reference needs to go up to 2.4.1?

Reply:
I doesnt break the code, so I see no reason why it should not be updated. Its just a minor release
All packages that had major releases has not been updated to newest